ngtools/utils.py

```diff
@@ -62,3 +65,25 @@ class bright:
     okgreen = fg.bright.green
     warning = fg.bright.yellow
     fail = fg.bright.red
+
+
+def neuroglancer_state_to_neuroglancer_url(
+        state,
+        base_url="https://neuroglancer.lincbrain.org/cloudfront/frontend/index.html"
+):
+    """
+    Takes the current State of the Neuroglancer Viewer and converts it into a url-encoded value for the browser
+
+    """
+    ordered_dict = state._json_data
```

Comment:
The neuroglancer state has a public `to_json()` method that may be more robust than accessing the private `_json_data` attribute.

Reply:
Good catch -- this simplified things -- updated